src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import exception.DukeException;
+import task.Task;
+
+
+public class ParserTest {
+
+    @Test
+    public void isValidDate_correctInput_test() {
```

Comment:
Maybe adding the outcome for the test in the name makes its purpose more clear? E.g. isValidDate_correctInput_returnTrue()